csrankings-l.csv

```diff
@@ -39,6 +39,7 @@ Lana Yarosh,University of Minnesota,http://lanayarosh.com,n6bjh24AAAAJ
 Lance Fortnow,Illinois Institute of Technology,http://www.cc.gatech.edu/people/lance-fortnow,Yf5wVssAAAAJ
 Lance R. Williams,University of New Mexico,https://www.cs.unm.edu/~williams,OJBW1aUAAAAJ
 Lance Williams,University of New Mexico,https://www.cs.unm.edu/~williams,OJBW1aUAAAAJ
+Landu Jiang,Shenzhen University,https://csse.szu.edu.cn/pages/user/index?id=1132,UIonkqUAAAAJ

```

Comment:
Apparently research faculty, not tenure-track? ("associate researcher") Please clarify. Only tenure-track faculty are eligible for inclusion.